assets/in

```diff
@@ -74,9 +74,6 @@ if [ "$skip_download" = "false" ]; then
     --entrypoint /tmp/print-metadata  \
     "$image_name" > ${destination}/metadata.json
 
-  mkdir -p ${destination}/rootfs/
-  docker export $(cat /tmp/container.cid) | tar -xf - -C ${destination}/rootfs/
```

Comment:
I'm going to close this as there's no real input here on how I might be able to proceed.